integration-test/test/basic.bats

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bats
+
+load test_helper
+
+@test "meta: docker is installed" {
+    run docker version
+    echo "$output">&2
+    [ "$status" -eq 0 ]
```

Comment:
Should you check status before running echo?  Won't that corrupt "run docker version", which I assume is the intent.